app/components/groups/group-edit/edit-group.controller.js

```diff
@@ -4,8 +4,36 @@
 
     .controller("EditGroupController", EditGroupController)
 
-    EditGroupController.$inject = ["$scope"]
+    EditGroupController.$inject = ["GroupResource", "$modal", "$scope",
+      "$state", "$stateParams"]
 
-    function EditGroupController($scope){
+    function EditGroupController(GroupResource, $modal, $scope, $state, $stateParams){
```

Comment:
I think you don't use `$state` here. And it's better to pass `some_id` instead of `$stateParams`